### ISSUE TITLE
Syntax error in 1.1.10

### ISSUE THREAD

**Issue by audrow** [nas5w/combinate#14]
After bumping from 1.1.9 to 1.1.10, I get the following new error when I run my program with `ts-node`:
```
/Users/audrow/Downloads/yatm/node_modules/combinate/index.ts:1
function combinate<O extends Record<string | number, any[]>>(obj: O) {
                  ^

SyntaxError: Unexpected token '<'
    at Object.compileFunction (node:vm:352:18)
    at wrapSafe (node:internal/modules/cjs/loader:1031:15)
    at Module._compile (node:internal/modules/cjs/loader:1065:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Object.require.extensions.<computed> [as .ts] (/usr/local/lib/node_modules/ts-node/src/index.ts:1361:43)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/audrow/Downloads/yatm/src/test-cases/generator/generate-test-cases.ts:1:1)
```
My `tsconfig.json` is as follows:
```
{
  "extends": "@tsconfig/node16/tsconfig.json",
  "compilerOptions": {
    "moduleResolution": "node",
    "outDir": "dist"
  },
  "include": ["src/**/*.ts"]
}
```
I'm not exactly sure why it's giving me the error, but reverting lets my code get back to a working state, so that's what I'm doing for now. Let me know if I'm doing something wrong.

**Comment by nas5w** [comment]
Thanks for reporting this! Let me see if I can reproduce the error and I'll get back to you.

**Comment by nas5w** [comment]
@audrow I just shipped 1.1.11, which I believe should fix the issue. I'd appreciate it if you let me know if this works!

I'm actually a bit surprised `ts-node` was working for earlier versions since it doesn't compile `node_modules` and I hadn't been shipping the compiled `dist` directory with the package (I am now).

**Comment by nas5w** [comment]
I believe this has now been resolved in 1.1.11, so I'm going to close the issue. Please let me know if it is still a problem!
